examples/README

```diff
@@ -0,0 +1,21 @@
+Build all examples using "make" or "make all"
+
+To build examples individually, enter "make" followed by the filename (omitting the
+file extension)
+
+To remove all the executables run "make clean"
+
+Requirments to build and run the examples:
+  * Have the SOS installation "bin" directory set in the environment PATH
+  * Have a compatible process laucher (like Hydra version 3.2) set in the
+    environment PATH
+  * An example of setting the environment PATH on bash:
+    $ export PATH=<path-to-SOS-install>/bin:<path-to-hydra-install>/bin:$PATH
+
+The hello world example can be run with 4 processes like so: 
+  oshrun -n 4 ./hello
```

Comment:
We have asked to add `hydra` in the PATH, but then using `oshrun` here. Might it be better to use `hydra` here?

Reply:
Since the Getting Started Guide encourages the use of `oshrun` to run the programs should I instead just make a note that `oshrun` invokes `hydra` and that is why it is needed in the `PATH`?

Reply:
Seems simpler to me to encourage using `oshrun` for these examples and referring to the getting started guide for more details about the launcher.  

The guide should be clear about what happens by invoking `oshrun` - i.e., it picks up the following launchers in this order of preference: yod, mpirun, mpiexec, and srun.  (But we only test/guarantee compatibility with Hydra.)  Also the user can set the `OSHRUN_LAUNCHER` env var for a custom launcher.

Reply:
ok, sounds good